.storybook/components/Theming/Theming.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import type { StoryObj } from '@storybook/react';
+import React from 'react';
+import { ThemingDocumentation } from './ThemingDocumentation';
+import ThemingDocs from '../../../docs/THEMING.md';
+
+export default {
+  title: 'Documentation/Theming',
+  component: ThemingDocumentation,
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    axe: {
+      skip: true,
+    },
+  },
+};
+
+type Args = React.ComponentProps<typeof ThemingDocumentation>;
+
+export const Theming: StoryObj<Args> = {
+  args: {
+    children: <ThemingDocs />,
```

Comment:
I haven't looked at these docs components before 🤔 
- Do we pass ThemingDocs into a layout component so it can be markdown?
- Can we use Theming.storires.mdx and get away with only having one file, instead of 2?

(just curious, so not blocking)

Reply:
Oo, good questions.

> Do we pass ThemingDocs into a layout component so it can be markdown?

I believe the layout components are just to add padding and restrict the line length.

> Can we use Theming.storires.mdx and get away with only having one file, instead of 2? https://storybook.js.org/docs/react/api/mdx

I collapsed them into the `.stories` file. Is this better? We still get the padding etc and that prism highlighting but with a little less boilerplate. https://github.com/chanzuckerberg/edu-design-system/pull/1385/commits/4406eda81d6614f5b1d5473c72784f024ded39d6

Reply:
Another thing I was thinking is taking the component with the layout and prism highlighting and make it generic so all the documentation pages can reuse it. I think that would be even better because it would reduce some repetition.

Reply:
Ticket to further refactor these documentation components: https://czi-tech.atlassian.net/browse/EFI-787

Reply:
A generic documentation layout component makes lots of sense 👍

Reply:
Or honestly writing the story like you did is great, as well